ProjectSidewalk/SidewalkWebpage
# Resolves #1276: Adds keyboard shortcuts for selecting label tags

Resolves #1276 

I've added keyboard shortcuts for selecting label tags. All you have to do is open a label, and press one of the underlined letters to toggle its corresponding tag. 

Here are some screenshots of the label screens:

![image](https://user-images.githubusercontent.com/12887715/51661479-d52a7c00-1f65-11e9-86b8-3e15c5667c27.png)
![image](https://user-images.githubusercontent.com/12887715/51661486-d9569980-1f65-11e9-97c3-b62a11c292c0.png)
![image](https://user-images.githubusercontent.com/12887715/51661496-dcea2080-1f65-11e9-90f7-498c5d92dcba.png)
![image](https://user-images.githubusercontent.com/12887715/51661502-e1163e00-1f65-11e9-95e1-73e28090faf4.png)
![image](https://user-images.githubusercontent.com/12887715/51661509-e4a9c500-1f65-11e9-8aeb-58621fd2cb5a.png)

To test:
- Ensure that all of the hotkeys toggle the tags as intended
- Ensure that the hotkeys do not interfere with other hotkeys we have (in case I missed something!)
- Ensure that everything else with labels works as before

I have performed these test on Chrome and Firefox on my machine and everything works fine!
